apps/mainsite/settings.py

```diff
@@ -295,8 +295,14 @@ def legacy_boolean_parsing(env_key, default_value):
             'class': 'logging.handlers.TimedRotatingFileHandler',
             'when': 'H',
             'interval': 1,
-            'backupCount': 30 * 24,  # 30 days times 24 hours
-            'filename': os.path.join(LOGS_DIR, 'badgr_debug.log'),
+            'backupCount': 30*24,  # 30 days times 24 hours
+            'filename': os.path.join(LOGS_DIR, 'badgr_debug.log')
+        },
+        'badgr_debug_console': {
```

Comment:
Kan het zijn dat deze met een rebase per ongeluk erin is gekomen? 

Volgens mij kan het geen kwaad, dus geen blocker, maar mss even dubbelchecken?

Reply:
Ja, morgen even samen naar kijken.